sanic/__main__.py

```diff
@@ -35,10 +35,10 @@
 
         app.run(host=args.host, port=args.port,
                 workers=args.workers, debug=args.debug, ssl=ssl)
-    except ImportError:
-        log.error("No module named {} found.\n"
+    except ImportError as e:
+        log.error("{} found.\n"
```

Comment:
I have to admit, it is confusing, the default message of `ImportError` is negative, the final output reads okay.

Anyway, since Sanic only works with Python 3.5+, we can use ImportError.name to report the missing module name, so I've pushed an improved version, please check the latest commit.